templates/admin-notices/no-longer-needed.php

```diff
@@ -16,22 +16,22 @@
 	</p>
 
 	<p>
-		<?php _e( 'Please keep in mind that only notices on the public facing part of your site, or if you have disabled the display on the back end, will be logged and accounted for.', 'enable-jquery-migrate-helper' ); ?>
+		<?php _e( 'Please keep in mind that the log and the reports include errors encountered on the site admin pages (back end) only if you have disabled immediate display of errors in the back end.', 'enable-jquery-migrate-helper' ); ?>
```

Comment:
I believe this line needs refining.

Currently, deprecations are logged in two (/three) scenarios:
- You are browsing the front-end of a site
- You Are browsing the back-end and have disabled immediate displaying of errors
- In the case of using the default jQuery version with WordPress 5.6, this needs to be explicitly turned on in the plugin settings (because WP throws a _lot_ of deprecations by it self).

Not quite sure how we could word this though 🤔

Reply:
You are browsing the front-end of a site → Someone... 
---
What if we instead of saying "disabled immediate..." think in terms of
"Deprecation notices that occur in back end are either shown immediately on screen (default) or registered in the log."